File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size(),i,j=0;
        sort(nums.begin(),nums.end());
        for(i=1;i<n;i++)
        {
            if(nums[i] != nums[j])
                j++;
            nums[j] = nums[i];
        }
        return j+1;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = nums.length;
        int i = 1,j = 0;
        while(i<n){
            if(nums[i]!=nums[j]){
                j++;
            }
            nums[j] = nums[i];
            i++;
        }
        return j+1;
    }
}